content/docs/gke/deploy/deploy-cli.md

```diff
@@ -132,72 +170,77 @@ Follow these steps to deploy Kubeflow:
-  available:
+## Access the Kubeflow user interface (UI)
 
-    ```
-    https://<KFAPP>.endpoints.<project-id>.cloud.goog/
-    ```
-   * It can take 20 minutes for the URI to become available.
-     Kubeflow needs to provision a signed SSL certificate and register a DNS 
-     name.
-   * If you own/manage the domain or a subdomain with 
-     [Cloud DNS](https://cloud.google.com/dns/docs/)
-     then you can configure this process to be much faster.
-     See [kubeflow/kubeflow#731](https://github.com/kubeflow/kubeflow/issues/731).
+Follow these steps to access the Kubeflow central dashboard:
 
-1. We recommend that you check in the contents of your **${KFAPP}** directory
-  into source control.
+1. Enter the following URI into your browser address bar. It can take 20
+  minutes for the URI to become available:
 
-## Understanding the deployment process
+  ```
+  https://<KFAPP>.endpoints.<project-id>.cloud.goog/
+  ```
+
+1. Follow the instructions on the UI to create a namespace. See the guide to 
+  [creation of profiles](/docs/other-guides/multi-user-overview/#automatic-creation-of-profiles).
 
-The `kfctl` deployment process includes by the following commands:
+Notes:
 
-* **init** - performs a one-time setup.
-* **generate** - creates configuration files defining the various resources.
-* **apply** - creates or updates the resources.
-* **delete** - deletes the resources.
+* It can take 20 minutes for the URI to become available.
+  Kubeflow needs to provision a signed SSL certificate and register a DNS 
+  name.
+* If you own or manage the domain or a subdomain with 
+  [Cloud DNS](https://cloud.google.com/dns/docs/)
+  then you can configure this process to be much faster.
+  See [kubeflow/kubeflow#731](https://github.com/kubeflow/kubeflow/issues/731).
 
-With the exception of `init`, all commands take an argument which describes the
-set of resources to apply the command to. This argument can be one of the
-following:
+## Understanding the deployment process
 
-* **platform** - all GCP resources; that is, anything that doesn't run on 
-  Kubernetes.
-* **k8s** - all resources that run on Kubernetes.
-* **all** - all GCP and Kubernetes resources.
+The `kfctl` deployment process includes the following commands:
+
+* **build** - (Optional) Creates configuration files defining the various 
+  resources in your deployment. You only need to run `kfctl build` if you want 
+  to edit the resources before running `kfctl apply`. See the guide to
+  [customizing your Kubeflow deployment](/docs/gke/customizing-gke).
+* **apply** - Creates or updates the resources.
+* **delete** - Deletes the resources.
 
 ### App layout
 
-Your Kubeflow app directory **${KFAPP}** contains the following files and directories:
+Your Kubeflow app directory **${KFAPP}** contains the following files and 
+directories:
 
 * **app.yaml** defines configurations related to your Kubeflow deployment.
```

Comment:
The YAML file will be whatever the user supplied to -f

Reply:
I'll address this when we know the answer to the previous comment, about whether this change is definitely in v0.7.

Reply:
The change will definitely be in 0.7; until it is 0.7 is broken.

Reply:
Sounds good. Done. We must make sure we don't publish the updated docs until this change is in, assuming we publish the docs for one of the RCs.